README.md

```diff
@@ -23,22 +23,50 @@ You can build the project with all supported [Mbed OS build tools](https://os.mb
+    1. Create an IoT policy
+     - Click on **IoT Core**
+      ![AWS_Core](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/step1-IoT_Core.jpg)
+     - On the left panel click on **Secure > Policies**
+      ![AWS_Core](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/secure_policies_pic2.jpg)
+     - Click on **Create**
+      ![Create Policies](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/step7-create-policy.jpg)
+     - Click on **Advanced mode**
+      ![Policies](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/creating_policy_click_advanced_mode.jpg)
+     - Fill in your policy as shown below then click **Create**
+      ![Policies](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/aws_policy_advanced_mode_example.jpg)
+      **note:** you will need to substitute the **Resource** value above with your actual value from your **Thing** (which we will create next) by editing the policy after registering your **Thing**.
+      
+    2. Register a **Thing** 
+     - Click on **Manage > Things > Create**
+      ![Things](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/step2-Things-Create.jpg)
+     - Click on **Create a single thing**
+      ![Create](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/step3-create-single-thing.jpg)
+     - Fill in registry 
+      ![step4](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/create_thing.jpg)
+      **note**: no need to specify a type
+      
+    3. Generate a device certificate and keys
+     - Click on **Create certificate**
+      ![step4](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/certificate_option.jpg)
+     - Download the public key, private key, and the certificate for this **Thing** then click on **Activate**
+      ![step4](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/download_certificates.jpg)
+     - Click on **Attach a policy**
+      ![step4](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/attach_policies.jpg)
+     - Ensure the correct policy is selected then click on **Register Thing**
+      ![step4](https://raw.githubusercontent.com/COTASPAR/AWS_base_guide/master/images/attach_policy_register_thing.jpg)
 
-## Configuring the AWS IoT Core service
-
-1. Create an AWS account if you don't have one, and login to it.
-
-    __NOTE:__ If you have an admin for your AWS account, please contact them to add a user to the account. You should obtain your login credentials from your admin in this case.
-
-1. Set up device credentials and policy via the AWS IoT console. You can refer to the AWS documentation [here](https://docs.aws.amazon.com/iot/latest/developerguide/iot-gs.html). Follow the steps there to
```

Comment:
This linked page has changed since we created this README initially. The new, useful one is https://docs.aws.amazon.com/iot/latest/developerguide/create-iot-resources.html - shall we simply point users to this new page.

Reply:
Yes, we should point to the new page.